rpcs3/Emu/Cell/Modules/cellSsl.cpp

```diff
@@ -23,14 +25,22 @@ s32 cellSslEnd()
 	return CELL_OK;
 }
 
-std::string getCert(const std::string certPath, const int certID)
+std::string getCert(const std::string certPath, const int certID, const bool isNormalCert = false)
 {
-	//Stupid cellssl
 	int newID = certID;
-	if (certID == 6)
-		newID = 23;
-	if (certID > 6 && certID < 23)
-		newID = certID - 1;
+
+	// The 'normal' certs have some special rules for loading.
+	if (isNormalCert)
```

Comment:
Cant we check if certID is in 6 to 23 range and don't do code block if it's not?
Eg. if (isNormalCert && certID >= BartimoreCert && certID <= GTECyberTrustGlobalCert)
{...

Reply:
Done.